src/components/Card/Card.scss

```
@@ -11,6 +11,7 @@
 
   @include page-content-when-not-fully-condensed {
     border-radius: var(--p-border-radius-wide, border-radius());
+    overflow: hidden;
```

```suggestion
    overflow: hidden;
```
As of right not it's recommended for the child content to have the border-radius or `overflow: hidden;`, shipping this would potentially break multiple card layouts in the admin.

---

Alright, will close. Thanks Alex :)